linker/Mono.Linker/Annotations.cs

```diff
@@ -56,6 +56,18 @@ public class AnnotationStore {
 		System.Xml.XmlWriter writer;
 		GZipStream zipStream;
 
+		public void Clear ()
```

Comment:
I think I'll close it and you can create a new one when you know why/how it will be used